validator/lib/validateGlb.js

```diff
@@ -11,7 +11,14 @@ module.exports = validateGlb;
 function validateGlb(glb) {
     var version = glb.readUInt32LE(4);
 
+    cp.execFile('./GLTF/exec/gltf_validator', ['./GLTF/glb_sample/Box.glb'], function(err, stdout, stderr){
+        if(err) {console.error(err);}    
+        console.log('stdout', stdout);
+        console.log('stderr', stderr);
+    });
```

Comment:
This should go after the version check. 

While Node functions take callbacks, we prefer to use Promises. Check out https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/compressTexture.js#L207-L221 for an example of wrapping a Promise around a callback.

`validateGlb` will need to return a promise, and likewise so will `validateB3dm`, which means most areas that call `validateB3dm` will need to expect a promise, and not just a message.

I had some problems running this on Windows initially. `gltf_validator` is called `gltf_validator.bat`. I think `spawn` will work better to ignore the extension. Also its probably best to reference an absolute path for the validator exe - `var gltfValidatorPath= path.join(__dirname, '../bin/gltf_validator);`

Does the gltf validator accept a buffer as input, or does it only take files? If only files, the `glb` passed into `validateGlb` will need to be saved to a temporary file before being passed in. Check out https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/getTempDirectory.js for a part-way example.